### PR TITLE
RELATED: BB-849 Previous period measure definition

### PR DESCRIPTION
**Pull request by martinmosko**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-typings-opensource-repository-check/8/ : SUCCESS in 8s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-unit-tests-zuul-docker-el7/11/ : SUCCESS in 57s

File: src/VisualizationObject.ts [hunk 1]
```diff
@@ -131,39 +148,57 @@ export namespace VisualizationObject {
...
+            || (filter as IVisualizationObjectNegativeAttributeFilter).negativeAttributeFilter !== undefined
+        );
     }
 
     export function isPositiveAttributeFilter(
         filter: VisualizationObjectAttributeFilter
-    ): filter is IVisualizationObjectPositiveAttributeFilter {
-        return (filter as IVisualizationObjectPositiveAttributeFilter).positiveAttributeFilter !== undefined;
+    ):
+        filter is IVisualizationObjectPositiveAttributeFilter {
```
Comment (xMort) [inline src/VisualizationObject.ts]: This should probably be on the previous line.

File: src/VisualizationObject.ts [hunk 2]
```diff
@@ -131,39 +148,57 @@ export namespace VisualizationObject {
...
+        return !isEmpty(filter)
+            && (filter as IVisualizationObjectPositiveAttributeFilter).positiveAttributeFilter !== undefined;
     }
 
     export function isAbsoluteDateFilter(
         filter: VisualizationObjectDateFilter
-    ): filter is IVisualizationObjectAbsoluteDateFilter {
-        return (filter as IVisualizationObjectAbsoluteDateFilter).absoluteDateFilter !== undefined;
+    ):
+        filter is IVisualizationObjectAbsoluteDateFilter {
```
Comment (xMort) [inline src/VisualizationObject.ts]: Here as well.

File: src/VisualizationObject.ts (hunk 1 above)
Comment (martinmosko) [reply src/VisualizationObject.ts]: done

File: src/VisualizationObject.ts (hunk 2 above)
Comment (martinmosko) [reply src/VisualizationObject.ts]: done

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-typings-opensource-repository-check/9/ : SUCCESS in 9s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-unit-tests-zuul-docker-el7/12/ : SUCCESS in 30s

**Comment by xMort** [comment]
lgtm

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-typings-merge-compliance-check/4/ : SUCCESS in 15s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-unit-tests-zuul-docker-el7/14/ : SUCCESS in 43s

**Pull request merged**